README.md

```diff
@@ -43,15 +43,14 @@ The following table describes the service resources available in a STAT API impl
 supports all three of the foundation specifications. Note that the 'Endpoint'
 column is more of an example in some cases.
 
-| Endpoint                       | Specified in   | Returns                       | Description  |
-| ------------------------------ | -------------- | ----------------------------- | ------------ |
-| `/`                            | Core           | [?]()                         | |
-| `/products`                    | Core           | [ProductsCollection](./product) | Figure out which constraints are available for which `product_id` |
-| `/orders`                      | Core           | [OrdersCollection](./order)     |  |
-| POST `/orders`                 | Core           | [Order](./order)                | Order a capture with a particular set of constraints |
-| `/products/{productId}`        | Core           | [Product](./product)                   | |
-| `/opportunities`               | Opportunities  | [OpportunitiesCollection](./opportunity)   | |
-| POST `/opportunities`          | Opportunities  | [OpportunitiesCollection](./opportunity)    | Explore the opportunities available for a particular set of constraints |
+| Endpoint                    | Specified in  | Accepts                                                      | Returns                                                      | Description                                                  |
+| --------------------------- | ------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `GET /`                     | Core          | -                                                            | [Landing Page](#landing-page)                                |                                                              |
+| `GET /products`             | Core          | -                                                            | [Products Collection](./product/README.md)                   | Figure out which constraints are available for which `product_id` |
+| `GET /products/{productId}` | Core          | -                                                            | [Product](./product/README.md)                               |                                                              |
+| `GET /orders`               | Core          | -                                                            | [Orders Collection](./order/README.md#order-collection)      |                                                              |
+| `POST /orders`              | Core          | [Order Request](./order/README.md#order-request)             | [Order Object](./order/README.md#order-pobject)              | Order a capture with a particular set of constraints         |
```

Comment:
I'm not sure that we need a distinction between an OrderRequest and an Order.

Reply:
That's how it is documented now and is not coming in new in this PR.

Reply:
oh very true. misread the merge